drivers/i2c/i2c_ite_enhance.c

```diff
@@ -966,7 +964,10 @@ static int i2c_enhance_transfer(const struct device *dev,
 #endif
 	/* Lock mutex of i2c controller */
 	k_mutex_lock(&data->mutex, K_FOREVER);
-
+#ifdef CONFIG_PM
```

Comment:
I think you can skip the guard, `pm_policy_state_lock_get` is a noop if `PM=n`

Reply:
The `#ifdef CONFIG_PM` guard has been removed. The same change has also been applied to `i2c_ite_it8xxx2.c`. However, in `i2c_ite_it51xxx.c`, the guard is retained because a local function is used. I’ve verified that the code still builds successfully even when `CONFIG_PM `is disabled.